packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -66,13 +59,13 @@ const CodeContainer = styled(Box)<CodeContainerProps>((props) => {
  * </Box>
  * ```
  */
-const Code = ({ children, size = 'medium', type = 'subtle' }: CodeProps): JSX.Element => {
+const Code = ({ children, size = 'small' }: CodeProps): JSX.Element => {
   return (
     <CodeContainer size={size} as={isPlatformWeb ? 'span' : undefined}>
       <BaseText
-        color={`surface.text.${type}.lowContrast`}
+        color="surface.text.subtle.lowContrast"
         fontFamily="code"
-        fontSize={size === 'large' ? 100 : 75}
+        fontSize={size === 'small' ? 75 : 100}
```

Comment:
nit: create some function that takes in `size`, has a switch case or if..elseif and returns 75 or 100 depending on the size? This way if we add another size in the future, we will have a cleaner implementation for it instead of just checking for `small`?

Reply:
done